src/Psalm/Internal/Analyzer/FunctionLikeAnalyzer.php

```diff
@@ -766,13 +766,13 @@ function (FunctionLikeParameter $p) {
             }
         }
 
-        $plugin_classes = $codebase->config->after_functionlike_checks;
+        $plugin_functions = $codebase->config->after_functionlike_checks;
```

Comment:
The original name actually matched the variable contents - it's a list of classes.

Reply:
Oh, you're right. I thought for a moment it designated the role of the variable's content. Thanks, I close this PR then